src/playlist/Length.cxx

```diff
@@ -21,7 +21,7 @@
 
 static SignedSongTime get_duration(const DetachedSong &song) {
 	const auto duration = song.GetDuration();
-	return duration.IsNegative() ? (SignedSongTime)0 : song.GetDuration();
+	return duration.IsNegative() ? (SignedSongTime)0 : duration;
```

Comment:
?

Reply:
As I get the song duration in the local variable there is no need to call the GetDuration function a second time.

Reply:
True, but it doesn't fix an integer overflow